pkg/startup/v6/data/master/etc/origin/node/node-config.yaml

```diff
@@ -33,6 +33,16 @@ kubeletArguments:
   - RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true
   max-pods:
   - "50"
+  image-gc-high-threshold: 
+  - "80"
+  image-gc-low-threshold: 
+  - "60"
+  maximum-dead-containers:
```

Comment:
All below will be deprecated in favour of `kubelet eviction` . But they are still here, so worth keeping.

Reply:
it is getting deprecated, but even in 1.15 k8s it's there, so it's not phased out yet.

Reply:
I think the image GC is also getting deprecated as well. Doc link: https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/#user-configuration